client/src/Auth/Register/register.js

```diff
@@ -60,7 +63,7 @@ const Register = (props) => {
                             onBlur={props.onBlur}
                             onChange={(e) => {props.handleChange('password', e)} }
                         />
-                        {focus === 'password' && errors.password && <p className="error-text">Passwords must be over 6 characters</p>}
+                        {focus === 'password' && errors.password && <p className="error-text">Passwords must be over 5 characters</p>}
```

Comment:
This should also use constant PASSWORD_LENGTH_MIN. This way when we change the minimum length, the message will stay correct.